tool/grind.dart

```diff
@@ -84,7 +84,8 @@ serveCustomBackend() {
     '-i',
     's,https://dart-services.appspot.com,${Platform.environment[backendVariable]},g',
     'build/scripts/main.dart.js',
-    'build/scripts/embed.dart.js'
+    'build/scripts/embed.dart.js',
+    'build/scripts/new_embed.dart.js'
```

Comment:
nit: would end with a trailing comma to help future diffs

Reply:
Done.